### PR TITLE
Don't redact URLs during notebook event logging

### PR DESCRIPTION
**Pull request by dadlerj**
## App preview:

- [Web](https://sg-web-notebook-event-urls.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-lpzxxzdzcl.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Review by slimsag — APPROVED** [review]

File: client/web/src/tracking/util.ts [hunk 1]
```diff
@@ -27,8 +27,11 @@ export function redactSensitiveInfoFromAppURL(url: string): string {
         return url
     }
 
-    // Ensure we do not leak repo and file names in the URL
-    sourceURL.pathname = '/redacted'
+    // Capture urls for notebook pages
+    if (!sourceURL.pathname.startsWith('/notebooks')) {
```
Comment (dadlerj) [inline client/web/src/tracking/util.ts:31]:
```suggestion
    if (sourceURL.pathname.startsWith('/notebooks')) {
        return url
    }
```

File: client/web/src/tracking/util.ts [hunk 2]
```diff
@@ -27,8 +27,11 @@ export function redactSensitiveInfoFromAppURL(url: string): string {
...
+        // Ensure we do not leak repo and file names in the URL
+        sourceURL.pathname = '/redacted'
+    }
```
Comment (dadlerj) [inline client/web/src/tracking/util.ts:34]:
```suggestion
```

File: client/web/src/tracking/util.ts [hunk 3]
```diff
@@ -27,8 +27,11 @@ export function redactSensitiveInfoFromAppURL(url: string): string {
         return url
     }
 
-    // Ensure we do not leak repo and file names in the URL
-    sourceURL.pathname = '/redacted'
+    // Capture urls for notebook pages
+    if (!sourceURL.pathname.startsWith('/notebooks')) {
+        // Ensure we do not leak repo and file names in the URL
+        sourceURL.pathname = '/redacted'
```
Comment (dadlerj) [inline client/web/src/tracking/util.ts:33]:
```suggestion
    sourceURL.pathname = '/redacted'
```

**Comment by dadlerj** [comment]
@slimsag I will let you merge this if it looks good to you!

**Pull request merged**